admin/class-admin.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/*
+Plugin Name: Patreon
+Plugin URI:
+Description: Stay close with the Artists & Creators you're supporting
+Version: 1.0
+Author: Ben Parry
```

Comment:
🤔

Reply:
Yep, that's the original author of our wordpress plugin.